security/auto-host-endpoints.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Protect Kubernetes nodes
```

Comment:
I didn't see this page in the sidebar - is there something missing to insert it in there?

Reply:
d418d0a